cg/meta/upload/error_handling.py

```diff
@@ -0,0 +1,21 @@
+import logging
+from functools import wraps
+
+from cg.exc import CgError
+from cg.services.file_delivery.deliver_files_service.exc import DeliveryTypeNotSupported
+
+LOG = logging.getLogger(__name__)
+
+
+def handle_delivery_type_errors(func):
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except DeliveryTypeNotSupported as error:
+            LOG.info("Delivery type not supported or no delivery needed. Skipping delivery.")
+            return
+        except Exception as error:
+            raise CgError(f"{error}") from error
```

Comment:
I don't think this is necessary. All `Exceptions` are caught in the `upload_all_completed_analyses` CLI function, do we need to translate this into a CGError?

Reply:
I also don't really see the benefit. In my mind, exceptions should be remapped to more descriptive errors and otherwise kept as is. Is this more descriptive?

Reply:
I removed it